prime-router/src/test/kotlin/CsvFileTests.kt

```diff
@@ -60,7 +60,7 @@ class CsvFileTests {
         }
```

Comment:
I only see a "happy path" test and a "no header" test in here. We should have a few tests that invoke filters and check the results as well. Am I missing them?

Reply:
There are some SmokeTests that I found that exercise the filters, however they also only flow through the `cli` so while they provide some confidence I think some follow up tickets for solidifying testing would be great.